packages/url/src/index.js

```diff
@@ -34,6 +34,51 @@ export function addQueryArgs( url, args ) {
 	return baseUrl + '?' + stringify( { ...query, ...args } );
 }
 
+/**
+ * Returns a single query argument of the url
+ *
+ * @param  {string} url URL
+ * @param  {string} arg Query arg name
+ *
+ * @return {Array|string} Query arg value.
+ */
+export function getQueryArg( url, arg ) {
+	const queryStringIndex = url.indexOf( '?' );
+	const query = queryStringIndex !== -1 ? parse( url.substr( queryStringIndex + 1 ) ) : {};
+
+	return query[ arg ];
+}
+
+/**
+ * Determines whether the URL contains a given query arg.
+ *
+ * @param  {string}    url URL
+ * @param  {...string} arg Query arg name
+ *
+ * @return {boolean} Whether or not the URL contains the query aeg.
+ */
+export function hasQueryArg( url, arg ) {
+	return typeof getQueryArg( url, arg ) !== 'undefined';
+}
+
+/**
+ * Removes arguments from the query string of the url
+ *
+ * @param  {string} url  URL
+ * @param  {string} args Query Args
+ *
+ * @return {string} Updated URL
+ */
+export function removeQueryArgs( url, ...args ) {
+	const queryStringIndex = url.indexOf( '?' );
+	const query = queryStringIndex !== -1 ? parse( url.substr( queryStringIndex + 1 ) ) : {};
+	const baseUrl = queryStringIndex !== -1 ? url.substr( 0, queryStringIndex ) : url;
+
+	args.forEach( ( arg ) => delete query[ arg ] );
```

Comment:
Guess we've not brought in Lodash yet as a dependency of this one, but good use-case for `omit`.

https://lodash.com/docs/4.17.10#omit

Reply:
Yeah I looked into this one but since it's a one-liner it shouldn't be necessary to add a dependency just for this use case.